easybuild/easyblocks/p/python.py

```diff
@@ -104,6 +104,26 @@ def configure_step(self):
                 line = re.sub(r"^#(\s*-L\$\(SSL\)/lib )", r"\1 -L$(SSL)/lib64 ", line)
                 sys.stdout.write(line)
 
+        tcl = get_software_root('Tcl')
+        tk = get_software_root('Tk')
+        if tcl and tk:
```

Comment:
People are free to use the system Tcl/Tk (e.g. by suing `--filter-deps`), which is fine.